README.md

```diff
@@ -33,12 +33,12 @@ You can generate `std4`'s documentation with
 
 ```text
 # if you're generating documentation for the first time
-> lake -Kdoc=on update
+> lake -R -Kdoc=on update
 ...
 # actually generate the documentation
-> lake -Kdoc=on build Std:docs
+> lake -R -Kdoc=on build Std:docs
 ...
-> ls build/doc/index.html
+> open build/doc/index.html
```

Comment:
`ls` is right here given the expected output. I believe this is only to demonstrate the file exists. The `open` command doesn't work on many platforms.
```suggestion
> ls build/doc/index.html
```

Reply:
Ok, I'll revert this